src/main/java/com/cincialert/cincialert/CinciAlertController.java

```diff
@@ -0,0 +1,22 @@
+package com.cincialert.cincialert;
```

Comment:
Created controller class to view html. Current webpage showed whitelabel error page.